posthog/api/routing.py

```diff
@@ -151,7 +151,8 @@ def parents_query_dict(self) -> Dict[str, Any]:
         return result
 
     def get_serializer_context(self) -> Dict[str, Any]:
-        return {**super().get_serializer_context(), **self.parents_query_dict}
+        add_organization_id = {"organization_id": self.organization_id} if self.organization_id else {}
```

Comment:
I needed organization_id in the serializer and assumed it's cheap to add here...